pdns/dnsproxy.cc

```diff
@@ -85,41 +88,38 @@ void DNSProxy::go()
   pthread_create(&tid,0,&launchhelper,this);
 }
 
-/** returns false if p->remote is not allowed to recurse via us */
-bool DNSProxy::sendPacket(DNSPacket *p)
-{
-  uint16_t id;
-  {
-    Lock l(&d_lock);
-    id=getID_locked();
-
-    ConntrackEntry ce;
-    ce.id       = p->d.id;
-    ce.remote = p->d_remote;
-    ce.outsock  = p->getSocket();
-    ce.created  = time( NULL );
-    ce.qtype = p->qtype.getCode();
-    ce.qname = p->qdomain;
-    ce.anyLocal = p->d_anyLocal;
-    ce.complete=0;
-    d_conntrack[id]=ce;
-  }
-  p->d.id=id^d_xor;
-  p->commitD();
-  
-  const string& buffer = p->getString();
-  
-  if(send(d_sock,buffer.c_str(), buffer.length() , 0)<0) { // zoom
-    L<<Logger::Error<<"Unable to send a packet to our recursing backend: "<<stringerror()<<endl;
-  }
-  (*d_resquestions)++;
-  return true;
-
-}
+extern TCPNameserver *TN;
```

Comment:
Not blocking but why do we need to add that definition?

Reply:
`tcpreceiver.hh` include also looks unused?

Reply:
Unsure why those were in there, possibly from some intermediate iteration. Removed now.